src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.ts

```diff
@@ -98,6 +99,31 @@ export class TaskListWrapperComponent implements OnInit {
     });
   }
 
+  /**
+   * On the return of the refreshed tasks, we throw up a refresh message.
+   */
+  public refreshTasks(): void {
+
+    this.getTasks().subscribe(result => {
+
+      this.tasks = result.tasks;
+      this.ref.detectChanges();
+
+      const message: InformationMessage = {
+        type: InfoMessageType.INFO,
+        message: InfoMessage.LIST_OF_AVAILABLE_TASKS_REFRESHED,
+      };
+
+      this.infoMessageCommService.addMessage(message);
+    });
+  }
+
+  public getTasks(): Observable {
```

Comment:
I think this might be better named as something like "performSearch()` as the current name is easy to confuse with `loadTasks()`.

Reply:
Agreed.:)